migrator/migrations/n_52_to_n_53_postgres_simple_access_scopes/migration_test.go

```diff
@@ -86,3 +92,63 @@ func (s *postgresMigrationSuite) TestSimpleAccessScopeMigration() {
 		s.Equal(simpleAccessScope, fetched)
 	}
 }
+
+func (s *postgresMigrationSuite) TestPermissionSetMigration() {
+	newStore := pgPermissionSetStore.New(s.postgresDB.Pool)
+	legacyStore, err := legacypermissionsets.New(s.legacyDB)
+	s.NoError(err)
+
+	// Prepare data and write to legacy DB
+	var permissionSets []*storage.PermissionSet
+	for i := 0; i < 200; i++ {
+		permissionSet := &storage.PermissionSet{}
+		s.NoError(testutils.FullInit(permissionSet, testutils.UniqueInitializer(), testutils.JSONFieldsFilter))
```

Comment:
As I see, `testutils.FullInit` will generate an ID with UUID V4. This will always test best case scenario. Maybe we should add in tests a few permission sets with IDs:
- just string values
- prefixed UUIDs
- special types, like `analyst`, etc.

This suggestion can be also applied to access scopes.